src/Microsoft.ApplicationInsights.AspNetCore/Implementation/TelemetryConfigurationOptionsSetup.cs

```diff
@@ -51,9 +52,9 @@ public void Configure(TelemetryConfiguration configuration)
 
             if (this.telemetryProcessorFactories.Any())
             {
-                foreach (Func<ITelemetryProcessor, ITelemetryProcessor> processorFactory in this.telemetryProcessorFactories)
+                foreach (ITelemetryProcessorFactory processorFactory in this.telemetryProcessorFactories)
```

Comment:
I think @karolz-ms and @pharring are using it.
I agree it is a good change, I just don't think it belongs in this "hotfix" release of bug fixes.